versioned_docs/version-v3.0/output-targets/custom-elements.md

```diff
@@ -14,7 +14,7 @@ contributors:
 
 The `dist-custom-elements` output target creates custom elements that directly extend `HTMLElement` and provides simple utility functions for easily defining these elements on the [Custom Element Registry](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry). This output target excels in use in frontend frameworks and projects that will handle bundling, lazy-loading, and custom element registration themselves.
 
-This target can be used outside of frameworks as well, if lazy-loading functionality is not required or desired. For using lazily loaded Stencil components, please refer to the [dist output target](/distribution).
+This target can be used outside of frameworks as well, if lazy-loading functionality is not required or desired. For using lazily loaded Stencil components, please refer to the [dist output target](../output-targets/dist.md).
```

Comment:
Another (non-blocking) case where the link differs between the versions of the docs - see https://github.com/ionic-team/stencil-site/pull/1022/files#diff-009a946cd7a1bed32d997de1a2f768193f39f0285964f590a32c392fa5cd3d15R17